nipy/algorithms/registration/groupwise_registration.py

```diff
@@ -244,8 +246,8 @@ def __init__(self,
         # The reference scan conventionally defines the head
         # coordinate system
         self.optimize_template = optimize_template
-        if not optimize_template and refscan is None:
-            self.refscan = REFSCAN
+        if not optimize_template and refscan == None:
```

Comment:
This should be `refscan is None` : https://www.python.org/dev/peps/pep-0008/#programming-recommendations